### PR TITLE
add private and public sections tags

### LINKED ISSUE #1124: Add more insider features
Add more insider features: e.g., tags and icons.

### PR DESCRIPTION
**Pull request by NicolasMassart**
## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change

Added insider features:
- meta plugin for entire directory tree meta setting
- add private and public network meta tags on whole sections using meta plugin
- add tag icons
- add section icons on private and public networks sections
- additionally hide feedback widget section index pages as they are not important enough to disturb the user with the widget

## Issue fixed

fixes #1124

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [x] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

## Preview

https://hyperledger-besu--1127.org.readthedocs.build/en/1127/

### CODE REVIEW

File: docs/private-networks/.meta.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+# Meta idems setting with yaml files is an insider only feature
+# see https://squidfunk.github.io/mkdocs-material/reference/#built-in-meta-plugin
+
+tags:
+  - Private Networks
```
Comment (alexandratran) [inline docs/private-networks/.meta.yml:6]:
```suggestion
  - private networks
```

File: docs/public-networks/.meta.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+# Meta idems setting with yaml files is an insider only feature
+# see https://squidfunk.github.io/mkdocs-material/reference/#built-in-meta-plugin
+
+tags:
+  - Public Networks
```
Comment (alexandratran) [inline docs/public-networks/.meta.yml:6]:
```suggestion
  - public networks
```

File: mkdocs-insider.yml [hunk 3]
```diff
@@ -10,18 +10,33 @@
...
+    tags_file: tags-index.md
 extra:
   # indicates to the templating system that the current build uses insider
   # used to display insider only features in templates and prevent build fail on the normal version.
   is_insider: true
-
+  # tags ID, used for insider tag icons, see https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#tag-icons
+  tags:
+    Private Networks: priv
+    Public Networks: pub
```
Comment (alexandratran) [inline mkdocs-insider.yml:28]:
```suggestion
    private networks: priv
    public networks: pub
```

**Review by alexandratran — COMMENTED** [review]
There are many pages on the public networks side that require the private networks tag as well. How should we go about adding those?

**Comment by NicolasMassart** [comment]
> There are many pages on the public networks side that require the private networks tag as well. How should we go about adding those?

You can add tags individually directly in the pages. Can you provide a page path that should have both so that I add it as an example here?

**Comment by alexandratran** [comment]
https://besu.hyperledger.org/en/latest/public-networks/how-to/use-besu-api/ and everything in that section. Also, as another example, https://besu.hyperledger.org/en/latest/public-networks/how-to/configuration-file/.

**Review by alexandratran — APPROVED** [review]
Added private network tags to applicable pages on the public networks side. LGTM!

**Pull request merged**